inst/templates/ad_adsl.R

```diff
@@ -100,6 +100,11 @@ adsl <- dm %>%
   # See also the "Visit and Period Variables" vignette
   # (https://pharmaverse.github.io/admiral/cran-release/articles/visits_periods.html#treatment_adsl)
   mutate(TRT01P = ARM, TRT01A = ACTARM) %>%
+  ## derive birth date (BRTHDT) ----
```

Comment:
Like in the vignette I would move the `BIRTHDT` derivation to the `AAGE` derivation.